pkgs/development/tools/boost-build/default.nix

```diff
@@ -39,6 +39,8 @@ stdenv.mkDerivation {
   postPatch = ''
     substituteInPlace src/build-system.jam \
     --replace "default-toolset = darwin" "default-toolset = clang-darwin"
+  '' + lib.optionalString (lib.versionAtLeast version "1.82") ''
```

Comment:
```suggestion
  '' + lib.optionalString (useBoost ? version && lib.versionAtLeast useBoost.version "1.82") ''
```

Reply:
Nicely done! Thanks for the suggestion!!